src/level.py

```diff
@@ -187,7 +189,7 @@ def reset(self):
         # plants
         self.soil_layer.update_plants()
 
-        self.sky.set_time(6, 0)  # set to 0600 hours upon sleeping
+        self.sky.set_time(0, 0)  # set to 0600 hours upon sleeping
```

Comment:
...why did you change that? The game is going to boot back at night now...

Reply:
Ah this was a test just to make sure I didn't screw up something and the darkening of the day was turned off. I forgot to change it back to 6.

Reply:
Okay then, I'll trust you to revert that change